pkg/controller/database/rdsinstance_test.go

```diff
@@ -166,6 +172,30 @@ func TestConnect(t *testing.T) {
 				cr: instance(),
 			},
 		},
+		"SuccessfulUseSA": {
```

Comment:
Nitpick; Should we write the full `ServiceAccount` so it's clearer what we're testing in the test output?

Reply:
For sure!